app/src/pages/playground/RunMetadataFooter.tsx

```diff
@@ -14,6 +14,8 @@ import {
 import { LatencyText } from "@phoenix/components/trace/LatencyText";
 import { TokenCount } from "@phoenix/components/trace/TokenCount";
 
+import { EditSpanAnnotationsDialog } from "../trace/EditSpanAnnotationsDialog";
```

Comment:
I should probably move this to /components rather than doing a cross page import

Reply:
yeah if its not too much of a lift i would try, not a huge deal though

Reply:
Moved